src/components/Theme/style.css

```diff
@@ -33,13 +33,9 @@
 }
 
 .content {
+  /* Content height is 100% - 96px (height of footer + height of navbar)  */
+  height: calc(100% - 96px);
```

Comment:
the height of the footer and navbar is actually specified in  `em` in this branch.

Reply:
also it's necessary to make value dependencies explicit, otherwise it's easy to break the links. this mean this value should be computed instead of copy paster

Reply:
That's what I thought, I was just wondering whether I should do it as part of this PR or open a separate one for it. Depends on priority.

Reply:
you are introducing that dependency, so it's part of this scope

Reply:
Agreed. I've added computation for height based on footer height and navbar height.